lib/zclient.h

```diff
@@ -459,6 +459,9 @@ struct zapi_nexthop {
 	uint8_t label_num;
 	mpls_label_t labels[MPLS_MAX_LABELS];
 
+	uint16_t tunneltype;
+	vni_t r_vni;
```

Comment:
Why not re-use labels field? Label and vni are the same in evpn nlri. Interpretation depends on encap.